tools/privacyidea-pip-update

```diff
@@ -85,9 +86,10 @@ def main():
         update()
     else:
         answer = False
+        res = 'n'
         while not answer:
-            res = raw_input("Do you really want to update your "
-                            "python environment and the DB schema? (y/N)")
+            res = input("Do you really want to update your "
```

Comment:
Hm, under Python 2 we shouldn't use ``input`` because it evaluates the input: https://code-maven.com/never-use-input-in-python2
We could use six to find the right input for us:
```
from six.moves import input
```
The question is if we expect anyone to run this under Python 2?

Reply:
fixed in b065a00f